Build/stylecop.json

```diff
@@ -4,7 +4,8 @@
         "namingRules": {
             "allowedHungarianPrefixes": [
                 "ui", "x", "y", "my", "ls", "am", "id", "ie", "vs", "up"
-            ]
+            ],
+            "tupleElementNameCasing": "camelCase"
```

Comment:
I am not overly fond of these change, callsites look worse.
Besides it seems like a noisy change

Reply:
Both styles were in use before, so this was just a consolidation.

Reply:
Pascal case at callsites feels more C#

Reply:
I went through again and confirmed that camelCase is the dominant style in the code base (by far). It follows the view that tuples are more like a bag of parameters than a type with named fields.